README.md

```diff
@@ -1,24 +1,187 @@
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Testing](#testing)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#author)
+- [🔭 Future Features](#features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 Potosi Tech Talent Blog `[Ruby On Rails]` <a name="about-project"></a>
 
-* How to run the test suite
+Welcome to the Potosi Tech Talent Blog Page Ruby on Rails app repository! This app powers the platform where tech enthusiasts can write and share insightful blog posts on various tech-related topics. Whether you're a seasoned developer or a curious learner, this app provides a space to engage with the community and contribute to meaningful discussions.
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+    <summary>Client</summary>
+    <ul>
+      <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+      <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    </ul>
+  </details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+   <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[MVC Pattern]** Follows the MVC(Model, View, Controller) pattern which allows it to be easy to manage
+- **[Authentication]** Counts with an authentication system using [Devise Gem](https://github.com/heartcombo/devise) which allows the user to have an unique account with its own email and password, and also allows the user to edit its own profile
+- **[Migration of Database using Rails method]** All the data storaged is actually being managed by a Database, and the migration of the database is being done using the rails method
+- **[Testing]** All the views are being tested using [RSpec](https://rspec.info/) and [Capybara](https://github.com/teamcapybara/capybara)
+- **[Linters]** It follows the rules of 2 very well knowed linters [Rubocop](https://rubocop.org/) & [Stylelint](https://stylelint.io/), which helps to keep the code clean and easy to read
+- **[Gitflow]** Used [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) as a workflow which allows to have a clean and organized version control
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Preview <a name="live-demo"></a>
+
+Will be implemented in the future -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you add some more details in your Getting Started section such as commands which we can run to make your app run on the local machine 👍